common/proto-options/version.options

```diff
@@ -0,0 +1,2 @@
+# Options for file common/cypherock-common/proto/version.proto
+core.AppVersionResultResponse.app_versions type:FT_STATIC max_count:17
```

Comment:
keep it with some buffer make it 20 or 25.

Reply:
updated